### ISSUE TITLE
Add support for tox 3.7.0

### ISSUE THREAD

**Issue by ivar-lazzaro** [tox-dev/tox-pipenv#59]
With the new tox version out, venv._getresolvedeps() was completely removed in favor of venv. get_resolved_dependencies.

As of now, the following error will raise when trying a tox run

```
py36 recreate: /app/.tox/py36
Traceback (most recent call last):
  File "/usr/local/bin/tox", line 11, in <module>
    sys.exit(cmdline())
  File "/usr/local/lib/python3.6/dist-packages/tox/session.py", line 47, in cmdline
    main(args)
  File "/usr/local/lib/python3.6/dist-packages/tox/session.py", line 54, in main
    retcode = build_session(config).runcommand()
  File "/usr/local/lib/python3.6/dist-packages/tox/session.py", line 467, in runcommand
    return self.subcommand_test()
  File "/usr/local/lib/python3.6/dist-packages/tox/session.py", line 590, in subcommand_test
    self.run_sequential()
  File "/usr/local/lib/python3.6/dist-packages/tox/session.py", line 597, in run_sequential
    if self.setupenv(venv):
  File "/usr/local/lib/python3.6/dist-packages/tox/session.py", line 509, in setupenv
    status = venv.update(action=action)
  File "/usr/local/lib/python3.6/dist-packages/tox/venv.py", line 222, in update
    self.hook.tox_testenv_install_deps(action=action, venv=self)
  File "/usr/local/lib/python3.6/dist-packages/pluggy/hooks.py", line 284, in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
  File "/usr/local/lib/python3.6/dist-packages/pluggy/manager.py", line 68, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File "/usr/local/lib/python3.6/dist-packages/pluggy/manager.py", line 62, in <lambda>
    firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,
  File "/usr/local/lib/python3.6/dist-packages/pluggy/callers.py", line 208, in _multicall
    return outcome.get_result()
  File "/usr/local/lib/python3.6/dist-packages/pluggy/callers.py", line 80, in get_result
    raise ex[1].with_traceback(ex[2])
  File "/usr/local/lib/python3.6/dist-packages/pluggy/callers.py", line 187, in _multicall
    res = hook_impl.function(*args)
  File "/usr/local/lib/python3.6/dist-packages/tox_pipenv/plugin.py", line 80, in tox_testenv_install_deps
    deps = venv._getresolvedeps()
AttributeError: 'VirtualEnv' object has no attribute '_getresolvedeps'
```

**Comment by tonybaloney** [comment]
Merged in #60

**Comment by tonybaloney** [comment]
Patch released in 1.9.0 https://pypi.org/project/tox-pipenv/1.9.0/

<blockquote><img src="https://pypi.org/static/images/twitter.c0030826.jpg" width="48" align="right"><div><img src="/static/images/favicon.6a76275d.ico" height="14"> PyPI</div><div><strong><a href="https://pypi.org/project/tox-pipenv/1.9.0/">tox-pipenv</a></strong></div><div>A pipenv plugin for tox</div></blockquote>

**Comment by sunjiyun26** [comment]
[root@localhost bin]# tox --version 
['--version']
3.13.2 imported from /usr/lib/python2.7/site-packages/tox/__init__.pyc
registered plugins:
    tox-pipenv-1.9.0 at /usr/lib/python2.7/site-packages/tox_pipenv/plugin.pyc
    tox-venv-0.4.0 at /usr/lib/python2.7/site-packages/tox_venv/hooks.pyc

with 1.9.0

the same issue
